03.bowling/bowling.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+scores = ARGV[0].split(',')
+
+shots = []
+NUMBER_IN_LASTFRAME = 19
```

Comment:
```suggestion
LAST_FRAME_NUMBER = 19
```

みたいにするといいかも？（前置詞は無くせるなら無くす、lastとframeで2単語なのでアンダーバーで区切る）

あと定数はファイルの先頭（マジックコメントの下）に書くことが多いですー

Reply:
ご指摘ありがとうございます！定数名と位置を修正しました。